modules/tinymce/src/core/main/ts/api/html/DomParser.ts

```diff
@@ -101,6 +101,20 @@ const DomParser = (settings?: DomParserSettings, schema = Schema()): DomParser =
     const textBlockElements = schema.getTextBlockElements();
     const specialElements = schema.getSpecialElements();
 
+    const unwrapInvalidNode = (node: AstNode, originalNodeParent: AstNode = node.parent): void => {
+      if (specialElements[node.name]) {
```

Comment:
Do we care that this is called "unwrap" but does both unwrap and remove depending?

Reply:
Maybe add a comment for remove case to match the unwrap case? Idk

Reply:
`removeOrUnwrapInvalidNode`?

Reply:
If we don't find that too wordy, I'd rather that, yes please.

Reply:
I think it's okay